### PR TITLE
Add "forms in last 30 days" to admin report

### PR DESCRIPTION
**Pull request by kaapstorm**
This change adds a "# Form Submissions in last 30 days" column to the [HQ admin report](https://www.commcarehq.org/hq/admin/domains/). ([FB 215956](http://manage.dimagi.com/default.asp?215956))

Hold off on merge -- not yet tested locally -- @snopoke, I probably have an ES :mag::grey_question: question for you tomorrow.

cc @TylerSheffels

### CODE REVIEW

File: corehq/apps/domain/calculations.py [hunk 1]
```diff
@@ -119,6 +119,23 @@ def forms(domain, *args):
...
+    then = json_format_datetime(now - timedelta(days=int(days)))
+    now = json_format_datetime(now)
+    q = {
+        'query': {
+            'range': {
+                'received_on': {'from': then, 'to': now}},
+            'filter': {
+                'and': ADD_TO_ES_FILTER['forms'][:]}}}
+    data = es_query(params={'domain.exact': domain}, q=q, es_index='forms', size=1)
+    return data['hits']['total'] if data.get('hits') else 0
```
Comment (emord) [inline corehq/apps/domain/calculations.py]: Would be better to use https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/forms.py

`FormES().domain(domain).submitted(gte=then)`

Comment (kaapstorm) [inline corehq/apps/domain/calculations.py]: That is excellent! Thanks @emord.

**Comment by emord** [comment]
Looks good to me, but there's a don't merge tag on it

**Comment by kaapstorm** [comment]
Yeah, it's not working on staging. Looking into it.

**Comment by emord** [comment]
Looks like it's showing up in the table.

Add it to /a/<domain>/settings/project/internal/calculations/ and you will be able to easily test the calculation without having to do weird things with the timing on the task that calculates those properties

**Comment by snopoke** [comment]
looks good, not mergable

**Comment by kaapstorm** [comment]
Hey @emord, thanks for the tip. The calculation seems to work -- it returns a number without erroring, and the number looks right. So how come the column on the report on staging still says "Not yet calculated"? What would cause that?

**Comment by snopoke** [comment]
The calculations get run once a day via a task in `domain/tasks.py`

**Comment by snopoke** [comment]
sorry, not domain/tasks but `corehq.apps.reports.tasks.
update_calculated_properties`

**Comment by kaapstorm** [comment]
Thanks @snopoke! It looks like it's working just fine. Dropping "hold off".

**Pull request merged**